src/api.rs

```diff
@@ -93,6 +93,24 @@ impl fmt::Display for Direction {
     }
 }
 
+/// Audio direction, playback or capture.
```

Comment:
Copy-paste problem, please update :)
I think the link to the Microsoft docs for role would fit in better here than at `get_default_device()`.

Reply:
Yes I agree